keps/sig-node/727-resource-metrics-endpoint/README.md

```diff
@@ -202,11 +202,11 @@ Alpha:
 
 Beta:
 
-- [ ] Modify the metrics server to consume the kubelet resource metrics endpoint 3 releases after it is added to the kubelet
+- [X] Modify the metrics server to consume the kubelet resource metrics endpoint 3 releases after it is added to the kubelet
```

Comment:
You need to update kep.yaml and ask for prod-readiness review

Reply:
Thanks, updated kep.yaml. Will ask for PRR once the questionnaire is completed.